bitnami/redis/values.yaml

```diff
@@ -1412,6 +1412,63 @@ metrics:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param metrics.livenessProbe define livenessProbe for metrics container
+  ##
+  ## @param metrics.livenessProbe define livenessProbe for metrics container
+  ##
```

Comment:
These lines are causing the CI/CD to fail when attempting to autogenerate the README.md tables, could you please remove them?